### PR TITLE
final about-us section

### LINKED ISSUE #4: About Us section
#Tasks
* make about us section 
* style the section

### LINKED ISSUE #16: Edit responsive in about section
# Tasks
edit responsive part in about us section

### PR DESCRIPTION
**Pull request by suhail-radwan**
Relates #16

### CODE REVIEW

**Review by reemsh2000 — DISMISSED** [review]

File: css/media.css [hunk 1]
```diff
@@ -63,10 +64,24 @@
...
+  .our-team-box img {
+    width: 250px;
+    height: 300px;
+>>>>>>> 2e332681d76bd84e0d5be6d9e46465e9289b3362
   }
 }
 
 @media screen and (max-width: 768px) {
+<<<<<<< HEAD
+=======
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 2]
```diff
@@ -63,10 +64,24 @@
...
+  .our-team-subcontainer {
+    width: 100%;
+  }
+  .our-team-box {
+    height: 350px;
+  }
+  .our-team-box img {
+    width: 250px;
+    height: 300px;
+>>>>>>> 2e332681d76bd84e0d5be6d9e46465e9289b3362
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 3]
```diff
@@ -103,6 +118,7 @@
     font-size: 9vw;
   }
 
+>>>>>>> main
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 4]
```diff
@@ -142,6 +185,8 @@
   .contact-info {
     width: 80%;
     font-size: 8px;
+<<<<<<< HEAD
+=======
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 5]
```diff
@@ -178,6 +223,7 @@
     align-items: center;
     margin-top: 15px;
     font-weight: 100;
+>>>>>>> main
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 6]
```diff
@@ -215,7 +261,21 @@
   .my-button {
     font-size: 4vw;
   }
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 2e332681d76bd84e0d5be6d9e46465e9289b3362
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 7]
```diff
@@ -215,7 +261,21 @@
...
+  .socialMedia_container_icons {
+    width: 50%;
+  }
+}
+<<<<<<< HEAD
+=======
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/media.css [hunk 8]
```diff
@@ -262,3 +322,12 @@
...
 }
+<<<<<<< HEAD
+  
+}
+
+>>>>>>> main
+=======
+>>>>>>> main
+=======
+>>>>>>> 2e332681d76bd84e0d5be6d9e46465e9289b3362
```
Comment (sara219) [inline css/media.css]: leftover from resolving conflict

File: css/style.css [hunk 9]
```diff
@@ -222,6 +228,14 @@ body {
...
 }
 
+<<<<<<< HEAD
+>>>>>>> main
+
+=======
+>>>>>>> 2e332681d76bd84e0d5be6d9e46465e9289b3362
+
+=======
+>>>>>>> main
```
Comment (sara219) [inline css/style.css]: leftover from resolving conflict

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by sara219 — CHANGES_REQUESTED** [review]
you push the changes without resolving the conflict 
and have some leftover from the previous  resolving conflict

**Review by sara219 — APPROVED** [review]

**Review by reemsh2000 — APPROVED** [review]

**Review by emanjaber — APPROVED** [review]

**Pull request merged**